build.sc

```diff
@@ -104,12 +104,12 @@ class chisel3CrossModule(val crossScalaVersion: String) extends CommonModule wit
     )
   }
 
-  object test extends Tests with TestModule.ScalaTest {
-    override def scalacPluginClasspath = m.scalacPluginClasspath
+  override def scalacOptions = T {
+    super.scalacOptions() ++ Agg(s"-Xplugin:${plugin.jar().path}", "-P:chiselplugin:genBundleElements")
```

Comment:
We can start generating Bundle elements on master for 3.6, but we should probably just do it by making it always on rather than using the argument. Regardless this is fine as long as it isn't backported to 3.5.x